pages/common/aws-sqs.md

```diff
@@ -0,0 +1,36 @@
+# aws sqs
+
+> CLI for AWS SQS -  provides simple queue for storing messages to share it between applications.
```

Comment:
You're describing the SQS services instead of the command itself.
```suggestion
> Create, delete, and send messages to queues for the AWS SQS service.
```